CalibPPS/TimingCalibration/test/DiamondSampicCalibrationHarvester_cfg.py

```diff
@@ -0,0 +1,51 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("harvester")
+
+process.load("FWCore.MessageService.MessageLogger_cfi")
+
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(-1) )
+
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('Configuration.EventContent.EventContent_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, '113X_dataRun3_Prompt_Candidate_2021_08_24_17_11_47')
```

Comment:
@ChrisMisan same here
Please use the versioned GT: `113X_dataRun3_Prompt_PPStestSampicPCL_v1` which is equivalent to the candidate.
In the future, when we will include the production tag for PPSDiamondSampicCalibration in the Prompt GT it will be possible to update the code to use the autoCond version.

Reply:
changed